api/server/router/image/image.go

```diff
@@ -44,6 +44,7 @@ func (r *imageRouter) initRoutes() {
 		router.Cancellable(router.NewPostRoute("/images/{name:.*}/push", r.postImagesPush)),
 		router.NewPostRoute("/images/{name:.*}/tag", r.postImagesTag),
 		router.NewPostRoute("/images/prune", r.postImagesPrune),
+		router.NewPostRoute("/images/{name:.*}/manifest", r.postImagesManifest),
```

Comment:
I made it POST because I assumed we're sending registry auth token for private repos. But since we send it in the header instead of the payload, is GET fine?